src/base/bittorrent/session.cpp

```diff
@@ -1042,7 +1042,7 @@ void Session::adjustLimits(lt::settings_pack &settingsPack)
                          , maxActive > -1 ? maxActive + m_extraLimit : maxActive);
 }
 
-void Session::applyBandwidthLimits(lt::settings_pack &settingsPack)
+void Session::applyBandwidthLimits(lt::settings_pack &settingsPack) const
```

Comment:
Could we transform it to free function?

Reply:
yes by adding second parameter, are you sure?

Reply:
Ok, leave it as is.